services/server/youtube.ts

```diff
@@ -1,12 +1,14 @@
-type PlayListType = {
+export type PlaylistType = {
   title: string;
   link: string;
-}[];
+};
+
+type PlaylistArray = PlaylistType[];
 
 export async function getYoutubePlaylist(
   playlistId: string,
   videosCount: number
-): Promise<PlayListType> {
+): Promise<PlaylistArray> {
```

Comment:
as discussed this func is returning an array of youtube video links and not an array of playlists

Reply:
Done